lib/ezcater_rubocop/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module EzcaterRubocop
-  VERSION = "0.57.4"
+  VERSION = "0.58.0.rc1"
```

Comment:
Do we still need the `rc1` here?

Reply:
I'll remove it before making the actual release. This was so that I could cut a new version with the latest configuration change and test it in other code bases.